ts/navigation/AppStackNavigator.tsx

```diff
@@ -178,6 +179,10 @@ export const AppStackNavigator = () => {
             name={IDPayOnboardingRoutes.IDPAY_ONBOARDING_MAIN}
             component={IDPayOnboardingNavigator}
           />
+          <Stack.Screen
+            name={ROUTES.IDPAY_OPERATIONS_LIST}
+            component={OperationsListScreen}
+          />
```

Comment:
Considering that this route is only accessible from within the initiative details screen, it should be part of `IDPAY_INITIATIVE_DETAILS`. Maybe adding another navigator inside `idpay/initiative/details` could make things more clear. What you think?